ui/src/app/views/resource-card/component.tsx

```diff
@@ -48,6 +48,16 @@ const ResourceCardMenu: React.FunctionComponent<{
             <MenuItem icon='pencil' onClick={onRenameNotebook}>Rename</MenuItem>
             <MenuItem icon='copy' onClick={onCloneResource}>Clone</MenuItem>
             <MenuItem icon='trash' onClick={onDeleteResource}>Delete</MenuItem>
+            {/*
+              This does not support both playground mode and jupyterLab yet,
+              that is a work in progress. We do not need to worry about that
+              here, because the menu will not open if you do not have write
+              access, and playground mode is currently only enabled if you do
+              not have write access.
+            */}
+            <MenuItem icon='grid-view' onClick={onOpenJupyterLabNotebook}>
```

Comment:
Per slack, should make this flag-controlled

Reply:
Working on that now, lots of meetings today, but I will get to it as soon as I can.